### DISCUSSION TITLE
Custom Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by theAdhocracy** [craftcms/ckeditor#313] [Ideas]
With Redactor, we had multiple custom keyboard shortcuts set up, particularly for custom style blocks. It would be great to get some kind of similar functionality for CKEditor, preferably controlled through a CKEditor Config Options panel.

Ideally, this would allow overwriting of existing shortcuts and introduction of new shortcuts, both for standard functionality (like the "inline code" button or "strikethrough" button) and "custom" functionality, like individual styles.

A suggested format might be:
```
return {
   keyboard: {
       shortcuts: [
           {
              keys: "Ctrl+Alt+C",
              command: "code"
           },
           {
              keys: "Ctrl+Shift+D",
              command: "strikethrough",
           },
           {
              keys: "Ctrl+Shift+C",
              command: {
                  style: "callout",
              },
           },
       ],
   },
}
```

**Comment by mattbloomfield** [comment]
Similarly, we are moving from Vizy which has slash commands to add blocks. This would be really convenient! 
<img width="679" height="121" alt="image" src="https://github.com/user-attachments/assets/0ae2e0a2-5ecd-4a2c-baa2-9807faf9ba6b" />
